frontend/app/src/store/blockchain/index.ts

```diff
@@ -1,340 +1,6 @@
-  };
-
-  const getBlockchainAccounts = (chain: string): BlockchainAccountWithBalance[] => get(blockchainAccounts)[chain] ?? [];
-
-  const getAddresses = (chain: string): string[] => get(addresses)[chain] ?? [];
-
-  const { getAssociatedAssetIdentifier } = useAssetInfoRetrieval();
-
-  const getAccountDetails = (
-    chain: string,
-    address: string,
-  ): {
-    assets: AssetBalance[];
-    liabilities: AssetBalance[];
-  } => {
-    const chainAssets = get(balances)[chain] ?? {};
-    const addressAssets = chainAssets[address];
-
-    if (addressAssets) {
-      const { assets, liabilities } = addressAssets;
-      return {
-        assets: Object.entries(get(mergeAssociatedAssets(assets, getAssociatedAssetIdentifier))).map(([asset, balance]) => ({ asset, ...balance })),
-        liabilities: !liabilities ? [] : Object.entries(get(mergeAssociatedAssets(liabilities, getAssociatedAssetIdentifier))).map(([asset, balance]) => ({ asset, ...balance })),
-      };
-    }
-
-    return {
-      assets: [],
-      liabilities: [],
-    };
-  };
-
-  const fetchAccounts = async (
-    payload: MaybeRef<BlockchainAccountRequestPayload>,
-  ): Promise<Collection<BlockchainAccountGroupWithBalance>> => new Promise((resolve) => {
-    resolve(sortAndFilterAccounts(
-      get(groups),
-      get(payload),
-      {
-        getAccounts(groupId: string) {
-          return blockchainAccountList(get(blockchainAccounts)).filter(account => account.groupId === groupId);
-        },
-        getLabel(address, chain) {
-          return get(addressNameSelector(address, chain));
-        },
-      },
-    ));
-  });
-
-  const fetchGroupAccounts = async (
-    payload: MaybeRef<BlockchainAccountGroupRequestPayload>,
-  ): Promise<Collection<BlockchainAccountWithBalance>> => new Promise((resolve) => {
-    const params = get(payload);
-    const newVar = blockchainAccountList(get(blockchainAccounts)).filter(account => account.groupId === params.groupId);
-    resolve(sortAndFilterAccounts(newVar, params, {
-      getLabel(address, chain) {
-        return get(addressNameSelector(address, chain));
-      },
-    }));
-  });
-
-  const accountTags = (address: string): ComputedRef<string[]> => computed(() => {
-    const account = blockchainAccountList(get(blockchainAccounts))
-      .find(account => getAccountAddress(account) === address);
-    return account?.tags ?? [];
-  });
-
-  return {
-    accounts,
-    accountTags,
-    addresses,
-    balances,
-    blockchainAccounts,
-    blockchainTotals,
-    fetchAccounts,
-    fetchGroupAccounts,
-    getAccountByAddress,
-    getAccountDetails,
-    getAccounts,
-    getAddressBalances,
-    getAddresses,
-    getBlockchainAccounts,
-    groups,
-    removeAccounts,
-    removeTag,
-    updateAccountData,
-    updateAccounts,
-    updateBalances,
-    updatePrices,
-  };
-});
+export const useBlockchainStore = defineStore('blockchain', () => ({
+  // TODO: remove
+}));
```

Comment:
Is there any reason we can't just remove this file?

Reply:
no I was keeping it until I removed all the references and then forgot good catch :)